From 043dcd60c2e6f2a7d9d01e89d28f956f738c9bbf Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Wed, 17 Mar 2021 22:45:36 +0100
Subject: [PATCH] Revert "Include common.h (and indirectly param.h) rather than
 just param.h to have BLASLONG available w/o circular dependencies"

This reverts commit 34753eaebb8b2ddbc256e9e996c1fb315396a2a0.
---
 getarch_2nd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/getarch_2nd.c b/getarch_2nd.c
index 53ecccf3..c390ef52 100644
--- a/getarch_2nd.c
+++ b/getarch_2nd.c
@@ -4,7 +4,7 @@
 #else
 #include "config_kernel.h"
 #endif
-#include "common.h"
+#include "param.h"
 
 int main(int argc, char **argv) {
 
-- 
2.29.2.windows.3

